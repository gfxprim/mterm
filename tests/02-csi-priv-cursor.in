2 2
\e[?25laa\nb
