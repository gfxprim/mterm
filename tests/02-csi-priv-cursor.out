 --
|aa|
|b |
 --
size 2x2 cursor 1x1
